version/version.go

```diff
@@ -49,17 +48,8 @@ type versionInfo struct {
 	GoVersion  string `json:"go"`
 }
 
-func (v versionInfo) String() string {
-	return fmt.Sprintf(`%s: %s
-git commit: %s
-build tags: %s
-%s`,
-		v.Name, v.Version, v.GitCommit, v.BuildTags, v.GoVersion,
-	)
-}
-
-func newVersionInfo() versionInfo {
-	return versionInfo{
+func New() Info {
```

Comment:
missing godoc comment
```suggestion
func NewInfo() Info {
```